packages/mui-material/src/Rating/Rating.js

```diff
@@ -156,7 +156,7 @@ const RatingIcon = styled('span', {
     transform: 'scale(1.2)',
   }),
   ...(ownerState.iconEmpty && {
-    color: (theme.vars || theme).palette.action.disabled,
+    color: theme.palette.action.active,
```

Comment:
```suggestion
    color: (theme.vars || theme).palette.action.active,
```

@zanivan This is actually needed to be [compatible with `CssVarsProvider`](https://mui.com/material-ui/experimental-api/css-theme-variables/overview/)! (and the default ThemeProvider at the same time)

Here: `theme.vars.palette.action.action` -> `var(--mui-palette-action-active)`